jdaviz/configs/specviz/plugins/unit_conversion/unit_conversion.py

```diff
@@ -76,23 +85,37 @@ def __init__(self, *args, **kwargs):
         # TODO [markers]: existing markers need converting
         self.spectrum_viewer.state.add_callback('x_display_unit',
                                                 self._on_glue_x_display_unit_changed)
+
         self.spectrum_viewer.state.add_callback('y_display_unit',
                                                 self._on_glue_y_display_unit_changed)
 
         self.spectral_unit = UnitSelectPluginComponent(self,
                                                        items='spectral_unit_items',
                                                        selected='spectral_unit_selected')
-        self.flux_or_sb_unit = UnitSelectPluginComponent(self,
-                                                         items='flux_unit_items',
-                                                         selected='flux_unit_selected')
+
         self.flux_or_sb = SelectPluginComponent(self,
                                                 items='flux_or_sb_items',
                                                 selected='flux_or_sb_selected',
                                                 manual_options=['Surface Brightness', 'Flux'])
 
+        self.flux_unit = UnitSelectPluginComponent(self,
+                                                   items='flux_unit_items',
+                                                   selected='flux_unit_selected')
+
+        self.sb_unit = UnitSelectPluginComponent(self,
+                                                 items='sb_unit_items',
+                                                 selected='sb_unit_selected')
+
     @property
     def user_api(self):
-        return PluginUserApi(self, expose=('spectral_unit', 'flux_or_sb', 'flux_or_sb_unit'))
+        if self.app.config == 'cubeviz':
+            return PluginUserApi(self, expose=('spectral_unit', 'flux_or_sb', 'flux_unit', 'sb_unit'))   # noqa
+        elif self.app.config == 'specviz' and not self.specviz_disabler:
```

Comment:
nitpick.  Just for consistency with the rest of the if-statements (although see other comment above if we can use `flux_or_sb` instead).

```suggestion
       if self.app.config == 'specviz' and not self.specviz_disabler:
```

Reply:
Will do! 

Regarding the 'specviz_disabler', I did initially write it with the combination of the` self.app.config` and `flux_or_sb`, the logic in the unit_conversion.py and vue file seemed to be less readable with all the conditions and additional. I don't love having the extra traitlet but it did seem like the simpler solution. I can add additional comments and rename the traitlet to make it more clear what its used for.